gravatar-quickeditor/src/test/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModelTest.kt

```diff
@@ -351,69 +351,8 @@ class AvatarPickerViewModelTest {
             )
             assertEquals(
                 avatarPickerUiState.copy(
-                    emailAvatars = emailAvatarsCopy.copy(
-                        avatars = buildList {
-                            add(uploadedAvatar)
-                            addAll(emailAvatarsCopy.avatars)
-                        },
-                    ),
-                    uploadingAvatar = null,
-                    scrollToIndex = null,
-                ),
-                awaitItem(),
-            )
-        }
-        verify { fileUtils.deleteFile(uri) }
-    }
-
-    @OptIn(ExperimentalCoroutinesApi::class)
-    @Test
-    fun `given avatar returned when avatar with the same id then uiState is updated`() = runTest {
```

Comment:
❓ This one is not needed now?

Reply:
This logic is no longer part of the ViewModel as it's handled by the AvatarStore/Repository, so I think this test is unnecessary.